Content.Shared/Cluwne/CluwneComponent.cs

```diff
@@ -0,0 +1,70 @@
+using Robust.Shared.Audio;
+using Content.Shared.Chat.Prototypes;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.Cluwne;
+
+[RegisterComponent]
+public sealed class CluwneComponent : Component
+{
+    /// <summary>
+    /// Giggle emote chances.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float GiggleChance = 0.2f;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float KnockChance = 0.05f;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float GiggleRandomChance = 0.08f;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float GiggleCooldown = 5;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float RandomGiggleAttempt = 5;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float LastGiggleCooldown = 0f;
+
+    /// <summary>
+    /// Cluwne Emotes
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    public string GiggleEmoteId = "Scream";
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public string TwitchEmoteId = "Twitch";
+
+    [DataField("emoteId", customTypeSerializer: typeof(PrototypeIdSerializer<EmoteSoundsPrototype>))]
+    public string? EmoteSoundsId = "Cluwne";
+
+    public EmoteSoundsPrototype? EmoteSounds;
```

Comment:
Delete.

Reply:
moved this to the system instead of component, not sure if this is correct but the emote wont work without it?